fbgemm_gpu/bench/histogram_binning_calibration_benchmark.py

```diff
@@ -14,7 +14,8 @@
 import torch
 from torch import Tensor
 
-logging.basicConfig(level=logging.DEBUG)
+logger: logging.Logger = logging.getLogger()
```

Comment:
These changes are weird. Are they part of IFU?

Maybe we want multiple dedicated PRs instead.

Reply:
> These changes are weird. Are they part of IFU?
> 
> Maybe we want multiple dedicated PRs instead.

This change is from the cherry-pick of the commit https://github.com/pytorch/FBGEMM/commit/c01bbb88a915e63f3b1763d4065382d09aa2e708

Reply:
Unfortunately, without this commit benchmarks outputs nothing. So we either keep it or bump upstream branch. @xinyazhang Could you also clarify what is "IFU"?

Reply:
Oh, IFU is "Integrate From Upstream"